recipes/bioconductor-struct/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.3.1" %}
+{% set name = "struct" %}
+{% set bioc = "3.13" %}
+
+package:
+  name: 'bioconductor-{{ name|lower }}'
+  version: '{{ version }}'
+source:
+  url:
+    - 'https://bioconductor.org/packages/{{ bioc }}/bioc/src/contrib/{{ name }}_{{ version }}.tar.gz'
+  md5: 200623f38fa71f08198d6e9e20ff22db
+build:
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  noarch: generic
+# Suggests: testthat, rstudioapi, rmarkdown, covr, BiocStyle, openxlsx, ggplot2, magick
+requirements:
+  host:
+    - 'bioconductor-s4vectors >=0.28.0,<0.29.0'
+    - 'bioconductor-summarizedexperiment >=1.20.0,<1.21.0'
+    - r-base >4.0
```

Comment:
Needs to be R 4.1 for Bioc 3.13 for all packages

Reply:
After discussing this we decided to stick with the specified version. We will use the conda build within bioc 3.12 but use the source from 3.13.  r-base needs pinning as follows: 'r-base >4.0,<4.1'